.github/workflows/security_scans.yml

```
@@ -0,0 +1,22 @@
+name: SAST Snyk Code
+on:
+  schedule:
+    - cron: '23 14 * * *'
```

> The pipeline will run once a week and the results will be sent to Ansible AAP Snyk organization.

This is trying to run daily at 14:23 UTC.

If this is supposed to run once a week, maybe something like..

```suggestion
    - cron: '23 14 * * 6'
```

---

Done